mod-infra/src/main/scala/at/forsyte/apalache/infra/tlc/config/TlcConfig.scala

```diff
@@ -64,11 +64,11 @@ object ConfigModelValue {
  *   the name of a model value
  */
 case class ConfigModelValue(name: String) extends ConfigConstExpr {
```

Comment:
What is the difference between `ConfigModelValue` and `ConfigStrValue` now? It looks like `ConfigStrValue` handles model values too.

Reply:
`ConfigModelValue` is what you get when TLC model values are parsed (e.g. `a` and `b` in `Producers = {a,b}`).
`ConfigStrValue` is what you get when a `"`-value is parsed (e.g. `"abc"` or `"1_OF_X"`)